src/svea_sensors/launch/localize.launch

```diff
@@ -33,19 +35,32 @@
     <!-- Start wheel encoders -->
     <group if="$(arg use_wheel_encoders)">
         <include file="$(find svea_sensors)/launch/wheel_odometry.launch">
-            <arg name="start_serial" value="false"/>
+            <arg name="start_serial" value="true"/>
```

Comment:
We usually start serial in top-level so this should be false. Maybe good to remove serial all together from `wheel_odometry.launch`.